### DISCUSSION TITLE
진행상황 보고 (masters-team-b-김유정)

### DISCUSSION THREAD

**Discussion by yujung7768903** [lablup/contribution-academy-2021#112] [Show and tell]
### 요약
1123번 이슈를 진행하면서 스크롤이 2개가 있는 것이 불필요하다고 생각되어 이슈를 생성했습니다.
기존에 진행하고 있던 web-ui 컴포넌트의 1123번 이슈를 해결하려 했지만 잘 해결되지 않았습니다.
그래서 새로 생성한 이슈를 먼저 해결하려고 시도해보고 있습니다.
<img width="400" alt="Screen Shot 2021-10-09 at 1 18 14 PM" src="https://user-images.githubusercontent.com/68562176/136643841-23e326eb-976f-4b0e-8de6-75c916c68f4d.png">

> 생성한 이슈 링크 : https://github.com/lablup/backend.ai-webui/issues/1139
### 이슈에 대한 문제 원인 파악 & 계획
mwc-dialog안에 card 클래스를 가진 div 엘리먼트가 있는데, mwc-dialog에도 스크롤이 있고 안에 있는 divd에도 스크롤이 생겨 중복이 된 것 같습니다.                 

<img width="718" alt="Screen Shot 2021-10-10 at 11 23 42 PM" src="https://user-images.githubusercontent.com/68562176/136699815-fea6c068-0bff-4ce1-95f0-f638a731216e.png">          

개발자 도구로 확인했을 때
class가 content-area인 요소의 속성 중 max-height 속성을 해제하거나 overflow-y와 overflow-x 속성을 모두 해제했을 때 아래와 같이 스크롤이 하나만 남게 되는 걸 확인했습니다.

<img width="662" alt="Screen Shot 2021-10-10 at 11 27 10 PM" src="https://user-images.githubusercontent.com/68562176/136699969-e97cb929-559f-4172-abfd-14db0a011427.png">

그런데 위 사진처럼 스크롤이 남게 되었을 때 dialog의 header가 fixed되지 않기 때문에 dialog를 닫으려면 스크롤을 맨 아래 또는 맨 위로 올려야합니다. 그 부분이 조금 불편하지 않을까 싶지만, 일단은 위처럼 스크롤을 하나만 남기는 방향으로 먼저 수정해보려합니다.

이 문제 뿐만 아니라, dialog와 관련해서 약관을 다보고 "해제"버튼을 누른 후 "개인정보보호정책"버튼을 눌렀을 때 스크롤이 맨 아래에 위치하기 때문에 개인정보보호정책을 읽으려면 스크롤을 맨 위로 올려야하는 불편함을 느꼈습니다. 현재 진행하고 있는 이슈 해결 후에 이 문제와 관련해서 이슈를 생성해볼까 생각중입니다.

**Comment by lizable** [comment]
@yujung7768903 I just reviewed your PR([webui#1142](https://github.com/lablup/backend.ai-webui/pull/1142)) and added comment.
Please check it when you are available. 😉
